spec/models/stripe_wrapper_spec.rb

```diff
@@ -1,27 +1,26 @@
 require 'spec_helper'
 
-describe StripeWrapper::Charge do
+describe StripeWrapper::Customer do
   describe ".create", :vcr do
     let(:token) { generate_stripe_token }
 
-    subject(:charge) do
-      StripeWrapper::Charge.create(
-        amount: 999,
+    subject(:customer_creation) do
+      StripeWrapper::Customer.create(
         token: token,
-        description: 'hey'
+        email: 'foo@bar.org'
       )
     end
 
     context "when card is valid" do
       let(:card_number) { '4242424242424242' }
 
-      it "succesfully charges the card" do
-        expect(charge.status).to eq(:success)
+      it "succesfully creates the customer" do
+        expect(customer_creation).to be_successful
       end
 
-      describe "successful?" do
-        it "returns true if the charge is successful" do
-          expect(charge.successful?).to eq(true)
+      describe "#id" do
+        it "returns the customer id" do
+          expect(customer_creation.id).to be_an_instance_of(String)
```

Comment:
Being a string and being a specific id are two different things. I would update the spec description to be more accurate.